From 4ed109f138a3e295720b1064be0ac816f081403a Mon Sep 17 00:00:00 2001
From: Stefan Riedmueller <s.riedmueller@phytec.de>
Date: Fri, 21 Jan 2022 15:25:17 +0100
Subject: [PATCH 2/2] Revert "MMFMWK-8918 [dmabufheaps] enable dmabuf heaps
 memory allocator."

This reverts commit d6ad337837085f8dc1ef29eae6844edbf3a3915f.

Signed-off-by: Stefan Riedmueller <s.riedmueller@phytec.de>
---
 gst-libs/gst/allocators/gstdmabuf.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/gst-libs/gst/allocators/gstdmabuf.c b/gst-libs/gst/allocators/gstdmabuf.c
index a91cbcce15ca..b19d6820695e 100644
--- a/gst-libs/gst/allocators/gstdmabuf.c
+++ b/gst-libs/gst/allocators/gstdmabuf.c
@@ -69,11 +69,9 @@ gst_dmabuf_mem_map (GstMemory * gmem, GstMapInfo * info, gsize maxsize)
 
 #ifdef HAVE_LINUX_DMA_BUF_H
   if (ret) {
-    struct dma_buf_phys dma_phys;
     if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
       GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
           g_strerror (errno), errno);
-    ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_PHYS, &dma_phys);
   }
 #endif
 
@@ -86,7 +84,6 @@ gst_dmabuf_mem_unmap (GstMemory * gmem, GstMapInfo * info)
   GstAllocator *allocator = gmem->allocator;
 #ifdef HAVE_LINUX_DMA_BUF_H
   struct dma_buf_sync sync = { DMA_BUF_SYNC_END };
-  struct dma_buf_phys dma_phys;
 
   if (info->flags & GST_MAP_READ)
     sync.flags |= DMA_BUF_SYNC_READ;
@@ -97,7 +94,6 @@ gst_dmabuf_mem_unmap (GstMemory * gmem, GstMapInfo * info)
   if (ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_SYNC, &sync) < 0)
     GST_WARNING_OBJECT (allocator, "Failed to synchronize DMABuf: %s (%i)",
         g_strerror (errno), errno);
-  ioctl (gst_fd_memory_get_fd (gmem), DMA_BUF_IOCTL_PHYS, &dma_phys);
 #else
   GST_WARNING_OBJECT (allocator, "Using DMABuf without synchronization.");
 #endif
@@ -161,8 +157,7 @@ gst_dmabuf_allocator_alloc (GstAllocator * allocator, gint fd, gsize size)
 {
   g_return_val_if_fail (GST_IS_DMABUF_ALLOCATOR (allocator), NULL);
 
-  return gst_fd_allocator_alloc (allocator, fd, size,
-      GST_FD_MEMORY_FLAG_KEEP_MAPPED);
+  return gst_fd_allocator_alloc (allocator, fd, size, GST_FD_MEMORY_FLAG_KEEP_MAPPED);
 }
 
 /**
-- 
2.25.1

